package.json

```diff
@@ -1,20 +1,24 @@
 {
-  "name": "raha-api",
-  "version": "1.0.0",
+  "name": "@raha/api",
```

Comment:
I am asking NPM to give us `@raha`, but if the person who owns it doesn't give it up (may take up to a month if they never respond) then I have `@raha.app`.